src/tests/functional/plugin/conformance/subgraphs_dumper/tests/test_models/model_5.hpp

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/add.hpp"
+#include "openvino/op/relu.hpp"
+#include "openvino/op/parameter.hpp"
+#include "openvino/op/result.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/split.hpp"
+#include "openvino/op/read_value.hpp"
+#include "openvino/op/assign.hpp"
+#include "matchers/subgraph/read_value_assign.hpp"
+
+class Model_5 {
+protected:
+    std::shared_ptr<ov::Model> model;
+
+public:
+    Model_5() {
+        //                        param_0
```

Comment:
read_value and assign is pair in all cases. So I can't find `read_value_1`...

Reply:
yes, it is negative test, here we have 1 good pair and 1 assign and the test extract_2 is checking, that only 1 model is found